app/app.js

```diff
@@ -147,5 +147,11 @@ if (window._slanger) {
 
 // Only do the react-router gear on pages we've designated
 window["initializeReactRouter"] = () => {
-  ReactDOM.render(Routes, document.getElementById('root'));
+  const rootElement = document && document.getElementById("root");
+
+  if (rootElement) {
+    ReactDOM.render(Routes, rootElement);
+  } else {
+    throw "No #root element to render the application to!";
+  }
```

Comment:
Ah, the things Flow makes us do.

I’d maybe make an `if !rootElement` shortcut and throw, so we just have an if and no else :)

Reply:
Oh, also, let’s throw an Error object rather than a string (you’ll take your eye out with that!)